FirebaseFunctions/Tests/SwiftIntegration/IntegrationTests.swift

```diff
@@ -0,0 +1,258 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+import FirebaseFunctions
+import FirebaseFunctionsTestingSupport
+import XCTest
+
+class IntegrationTests: XCTestCase {
+  let functions = FunctionsFake(
+    projectID: "functions-integration-test",
+    region: "us-central1",
+    customDomain: nil,
+    withToken: nil
+  )
+  let projectID = "functions-swift-integration-test"
+
+  override func setUp() {
+    super.setUp()
+    functions.useLocalhost()
+  }
+
+  func testData() {
+    let expectation = self.expectation(description: #function)
+    let data = [
+      "bool": true,
+      "int": 2 as Int32,
+      "long": 9_876_543_210,
+      "string": "four",
+      "array": [5 as Int32, 6 as Int32],
+      "null": nil,
+    ] as [String: Any?]
+    let function = functions.httpsCallable("dataTest")
+    XCTAssertNotNil(function)
+    function.call(data) { result, error in
+      do {
+        XCTAssertNil(error)
+        let data = try XCTUnwrap(result?.data as? [String: Any])
+        let message = try XCTUnwrap(data["message"] as? String)
+        let long = try XCTUnwrap(data["long"] as? Int64)
+        let code = try XCTUnwrap(data["code"] as? Int32)
+        XCTAssertEqual(message, "stub response")
+        XCTAssertEqual(long, 420)
+        XCTAssertEqual(code, 42)
+        expectation.fulfill()
+      } catch {
+        XCTAssert(false, "Failed to unwrap the function result")
```

Comment:
I'm curious if it is worth printing the error here or `XCTUnwrap` provides enough information in place?

Reply:
It can only help to print. Done!